commit b1dbc9468aa4a872554a581c649717d60c3593d6 (refs/changes/45/3145/1)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-01-02T23:51:51+13:00 (1 year, 9 months ago)
    
    PUBAPI-1473: machines.73.test.js no longer match recent vmapi changes

diff --git a/test/machines.73.test.js b/test/machines.73.test.js
index e38a404..6028350 100644
--- a/test/machines.73.test.js
+++ b/test/machines.73.test.js
@@ -159,14 +159,14 @@ test('networks: invalid formats', function (tt) {
     tt.test('ipv4_count: wrong type', function (t) {
         provisionWithInvalidNetwork(t, [
             { ipv4_uuid: PROVISIONABLE_NET_UUID, ipv4_count: 'a' }
-        ], 'Invalid VM parameters: Invalid networks');
+        ], 'Invalid VM update parameters: Invalid networks');
     });
 
 
     tt.test('ipv4_count: wrong type', function (t) {
         provisionWithInvalidNetwork(t, [
             { ipv4_uuid: PROVISIONABLE_NET_UUID, ipv4_count: 2 }
-        ], 'Invalid VM parameters: Invalid networks');
+        ], 'Invalid VM update parameters: Invalid networks');
     });
 
 });
